ceph.spec.in

```diff
@@ -716,6 +716,7 @@ Group:		System/Libraries
 %if 0%{?rhel} || 0%{?fedora}
 Obsoletes:	ceph-libs < %{_epoch_prefix}%{version}-%{release}
 %endif
+Requires:	ceph-common = %{_epoch_prefix}%{version}-%{release}
```

Comment:
ceph-common requires librados2 on line 402, and here librados2 requires ceph-common, is this mutual requirements valid?